docs/core/preview3/tools/dotnet-build.md

```diff
@@ -23,33 +23,37 @@ dotnet-build -- Builds a project and all of its dependencies
 
-The `dotnet build` command builds multiple source file from a source project and its dependencies into a binary. 
-By default, the resulting binary is in Intermediate Language (IL) and has a DLL extension. 
-`dotnet build` also drops a `*.deps` file which outlines what the host needs to run the application.  
+Building requires the existence of an asset file (a file that lists all of the dependencies of your application), which means that you have to run [`dotnet restore`](dotnet-restore.md) prior to building the project. Lack of the assets file manifests as the inability of the tooling to resolve reference assemblies. 
 
-Building requires the existence of an asset file (a file that lists all of the dependencies of your application), which 
-means that you have to run [`dotnet restore`](dotnet-restore.md) prior to building your code.
+`dotnet build` uses MSBuild to build the project, thus it support both parallel builds and incremental builds. Please refer to [MSBuild documentation]() to get more information on those topics. 
 
-Before any compilation begins, the `build` verb analyzes the project and its dependencies for incremental safety checks.
-If all checks pass, then build proceeds with incremental compilation of the project and its dependencies; 
-otherwise, it falls back to non-incremental compilation. Via a profile flag, users can choose to receive additional 
-information on how they can improve their build times.
+In addition to its options, the `dotnet build` command will accept MSBuild options as well, such as `/p` for setting properties or `/l` to define a logger. You can find out more about these properties in the [`dotnet msbuild`](dotnet-msbuild.md) command documentation. 
+
+Whether the project is executable or not is determined by the `<OutputType>` property in the project file. The following example shows a project that will produce executable code: 
 
-In order to build an executable application instead of a library, you need to set the `<OutputType>` property:
 
 ```xml
 <PropertyGroup>
     <OutputType>Exe</OutputType>
 </PropertyGroup>
 ```
 
+In order to produce a library, simply omit that property. The main difference in output is that the IL DLL for a library will not contain any entry points and it will not be possible to execute it. 
```

Comment:
simply omit that property -> set the `<OutputType>` property to Library or simply omit it, since it's the default output.

it will not be possible to execute it.  -> it will not be executable